recipes/vowpalwabbit/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "vowpalwabbit" %}
+{% set version = "8.7.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://scgrahamarchive.blob.core.windows.net/vowpal-wabbit/vowpal_wabbit_2ee2bcf_rapidjson.tgz 
+  sha256: befed5ecd42decd533b8e11a98f0e76f9d48951c085b52786277d86ec06923ac
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake>=3.13
+  host:
+    - py-boost
+    - pip
+    - python
+  run:
+    - py-boost
+    - python
+
+test:
+  imports:
+    - vowpalwabbit
+  requires:
+    - numpy
+    - pandas
+    - pytest
+    - scipy
+    - scikit-learn
```

Comment:
Are none of these strict install requirements?

Reply:
they aren't strict requirements, one can use the package without them. however there is a sklearn wrapper included which does require sklearn and scipy, and the tests for this wrapper involve pandas

Reply:
OK!